tools/devops/automation/templates/sign-and-notarized/setup.yml

```diff
@@ -14,10 +14,11 @@ parameters:
 steps:
 
 # DO NOT USE THE checkout.yml template. The reason is that the template changes the hash which results in a problem with the artifacts scripts
-- checkout: self          # https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema#checkout
-  clean: true             # Executes: git clean -ffdx && git reset --hard HEAD
-  submodules: recursive
-  path: s/xamarin-macios
+- template:  sdk-unified/steps/checkout/v1.yml@templates
```

Comment:
Should probably remove the comment on line 16 that says not to use the checkout template if this is desired

Reply:
Unless I am reading this wrong and the comment is referring to a completely different checkout yaml template?

Reply:
It is talking about a completely diff one.